pbm/pitr/pitr.go

```diff
@@ -155,6 +159,36 @@ func (s *Slicer) Catchup() error {
 	return nil
 }
 
+func (s *Slicer) UncheckedCatchup() (err error) {
+	s.l.Debug("unchecked catchup")
+
+	defer func() {
+		if err == nil {
+			s.l.Debug("lastTS set to %v %s", s.lastTS, formatts(s.lastTS))
+		}
+	}()
+
+	chnk, err := s.pbm.PITRLastChunkMeta(s.rs)
+	if err != nil {
+		return errors.Wrap(err, "get last slice")
+	}
+
+	if chnk != nil {
+		s.lastTS = chnk.EndTS
```

Comment:
If the last chunk exists but too far back in the past, slicing will fail with an "insufficient range" error.
The range should be checked and the `s.lastTS` set to "now" if it's insufficient.